.github/workflows/compile.yml

```diff
@@ -104,6 +104,43 @@ jobs:
         run: |
           pip install -U esptool
 
+      - name: Setup platform.txt
+        run: |
+          baseFolder="$HOME/.arduino15/packages/m5stack/hardware/esp32"
+          
+          # Find the latest version of the folder
+          latestVersion=$(find "$baseFolder" -maxdepth 1 -type d -exec basename {} \; | sort -V | head -n 1)
+          
+          if [ -z "$latestVersion" ]; then
+              echo "No version found."
+              read -p "Press Enter to exit."
+              exit 1
+          fi
+          
+          # Full path to the file to be edited
+          file="$baseFolder/$latestVersion/platform.txt"
+          backupFile="$baseFolder/$latestVersion/platform.txt.bkp"
+          prefix="build.extra_flags.esp32"
```

Comment:
Hey @bmorcelli, Thanks for working on this change. I was looking at the code for deauth. I was just thinking, instead of messing around with platform.txt, if we could update the arduino-cli command on Line No. 157, it would be more readable and maintainable.

e.g.
```
arduino-cli compile --fqbn ${{ matrix.board.fqbn }} -e \
                              --build-property build.partitions=huge_app \
                              --build-property upload.maximum_size=3145728 \
                              --build-property compiler.cpp.extra_flags="${extra_flags}" \
                              --build-property build.extra_flags.esp32=" -w " \
                              --build-property compiler.c.elf.libs.esp32=" -zmuldefs " \
                              ./m5stick-nemo.ino
```

I haven't tested this change, Just thinking if this works then we can do this.

Reply:
I Was looking Marauders Actions, trying to understand how he modified this platform.txt file... he did this:
```
 - name: Modify platform.txt
        run: |
          echo "Chicken"
          for i in $(find /home/runner/.arduino15/packages/esp32/hardware/esp32/ -name "platform.txt"); do
            sed -i 's/compiler.c.elf.libs.esp32c3=/compiler.c.elf.libs.esp32c3=-zmuldefs /' "$i"
            sed -i 's/compiler.c.elf.libs.esp32s3=/compiler.c.elf.libs.esp32s3=-zmuldefs /' "$i"
            sed -i 's/compiler.c.elf.libs.esp32s2=/compiler.c.elf.libs.esp32s2=-zmuldefs /' "$i"
            sed -i 's/compiler.c.elf.libs.esp32=/compiler.c.elf.libs.esp32=-zmuldefs /' "$i"
            cat "$i" | grep compiler.c.elf.libs.esp32c3
            cat "$i" | grep compiler.c.elf.libs.esp32s3
            cat "$i" | grep compiler.c.elf.libs.esp32s2
            cat "$i" | grep compiler.c.elf.libs.esp32
          done
```

I is far more readable and easy to understand.. we just need to change the base folder to the M5Stack current version folder, so we can try this sample:

```
 - name: Setup platform.txt
        run: |
          for i in $(find /home/runner/.arduino15/packages/m5stack/hardware/esp32/ -name "platform.txt"); do
            sed -i 's/compiler.c.elf.libs.esp32c3=/compiler.c.elf.libs.esp32c3=-zmuldefs /' "$i"
            sed -i 's/compiler.c.elf.libs.esp32s3=/compiler.c.elf.libs.esp32s3=-zmuldefs /' "$i"
            sed -i 's/compiler.c.elf.libs.esp32s2=/compiler.c.elf.libs.esp32s2=-zmuldefs /' "$i"
            sed -i 's/compiler.c.elf.libs.esp32=/compiler.c.elf.libs.esp32=-zmuldefs /' "$i"
            cat "$i" | grep compiler.c.elf.libs.esp32c3
            cat "$i" | grep compiler.c.elf.libs.esp32s3
            cat "$i" | grep compiler.c.elf.libs.esp32s2
            cat "$i" | grep compiler.c.elf.libs.esp32
          done
```